scrapers_next/sd/committees.py

```diff
@@ -0,0 +1,57 @@
+from spatula import JsonListPage, JsonPage, SkipItem, URL
+from openstates.models import ScrapeCommittee
+
+
+class CommitteeList(JsonListPage):
+
+    source_string = "https://sdlegislature.gov/api/SessionCommittees/Session/68"
+    source = URL(source_string)
+
+    # I don't know if there's a method on the base class for this to be overridden
+    def standardize_chamber(self, original_chamber_text):
+        chamber_conversion = { "H":"lower", "S":"upper", "J":"legislature" }
+        return chamber_conversion[original_chamber_text] 
+
+    def process_item(self, item):
+        committee_json = item["Committee"]
+
+        # The Full House & Senate are included in the committe json list, tagged with the following property
+        if committee_json["FullBody"]:
+            self.skip(f"Not a committee: {committee_json['Name']}")
+
+        com_id = item["SessionCommitteeId"]
+        detail_link = f"https://sdlegislature.gov/api/SessionCommittees/Detail/{com_id}"
+        homepage = f"https://sdlegislature.gov/Session/Committee/{com_id}/Detail"
+
+        chamber = self.standardize_chamber(committee_json["Body"])
+        if chamber is None:
```

Comment:
Typically `if not chamber:` is more pythonic. Unless we are specifically looking for `None`.